--- /home/eugene/anaconda3/envs/zdlp_env/lib/python3.8/site-packages/textract/parsers/utils.py	2020-10-14 22:46:18.912349761 +0500
+++ utils.py	2020-12-06 19:00:10.000000000 +0500
@@ -71,6 +71,10 @@
     `Fabric <http://www.fabfile.org/>`_-like behavior.
     """
 
+    def __init__(self):
+        # due to python issue 8557 (can't search in PATH on Win without Shell)
+        self.need_shell = True if os.name == 'nt' else False 
+
     def run(self, args):
         """Run ``command`` and return the subsequent ``stdout`` and ``stderr``
         as a tuple. If the command is not successful, this raises a
@@ -82,6 +86,7 @@
             pipe = subprocess.Popen(
                 args,
                 stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                shell=self.need_shell
             )
         except OSError as e:
             if e.errno == errno.ENOENT:
